### DISCUSSION TITLE
代理问题

### DISCUSSION THREAD

**Discussion by Earlesscat** [Richasy/Bili.Uwp#573] [想法]
我的计算机安装了代理，在代理开启的时候不知道这个程序是自动检测代理还是什么原理，只要开了代理整个程序的缩略图都无法打开，能否请开发者考虑新增一个设置让用户选择是直接连接还是选择走代理联网？谢谢开发者

**Comment by ghost** [comment]
https://github.com/Fndroid/clash_for_windows_pkg/issues/180
我用clash开启uwp的本地回环就可以联网了
